docs/durable_execution/temporal.md

```diff
@@ -188,15 +188,14 @@ As the streaming model request activity, workflow, and workflow execution call a
 
 [`Agent.run(model=...)`][pydantic_ai.agent.Agent.run] normally supports both model strings (like `'openai:gpt-5.2'`) and model instances. However, `TemporalAgent` does not support arbitrary model instances because they cannot be serialized for Temporal's replay mechanism.
 
-To use different models at runtime, you can:
+To use model instances with `TemporalAgent`, you need to pre-register them by passing a dict of model instances to `TemporalAgent(models={...})`. You can then reference them by name or by passing the registered instance directly. If the wrapped agent doesn't have a model set, the first registered model will be used as the default.
 
-1. **Pre-register model instances**: Pass a dict of model instances to `TemporalAgent(models={...})`, then reference them by name or by passing the registered instance directly. If the wrapped agent doesn't have a model set, the first registered model will be used as the default.
-2. **Use model strings**: Model strings work as expected.
-3. **Use a provider factory**: For scenarios where you need to customize the provider (e.g., inject API keys from deps), pass a `provider_factory` to `TemporalAgent`.
+Model strings work as expected. For scenarios where you need to customize the provider used by the model string (e.g., inject API keys from deps), you can pass a `provider_factory` to `TemporalAgent`.
```

Comment:
... which is passed the run context (with a link). I liked the explicit mention you had of getting API keys from deps